src/runtime-boot/init/src/main.rs

```diff
@@ -4,30 +4,47 @@ extern crate serde_json;
 
 use libc::syscall;
 
+use anyhow::{anyhow, Result};
+use nix::mount::MsFlags;
 use std::error::Error;
+use std::ffi::CString;
+use std::fs;
 use std::fs::File;
 use std::io::prelude::*;
 use std::io::{ErrorKind, Read};
 
+use anyhow::{anyhow, Result};
 use std::ffi::CString;
+use std::fs;
 use std::mem::size_of;
+use std::path::{Path, PathBuf};
 
 fn main() -> Result<(), Box<dyn Error>> {
-    // TODO: Get the rootfs key and other parameters through RA/LA or PAL
-    let rootfs_key = b"c7-32-b3-ed-44-df-ec-7b-25-2d-9a-32-38-8d-58-61";
     let rootfs_upper_layer = "/sefs/upper";
     let rootfs_lower_layer = "/sefs/lower";
     let rootfs_entry = "/";
 
-    // Get the key of FS image if needed
+    // Create and mount directory for fs key from agent enclave
+    fs::create_dir("/mnt");
+    let fs_type = String::from("hostfs");
+    let source = Path::new("/host");
+    let mount_path = Path::new("/mnt");
+    let flags = MsFlags::empty();
+    let source_c = CString::new(source.to_str().unwrap()).unwrap();
+    let mountpoint_c = CString::new(mount_path.to_str().unwrap()).unwrap();
+    nix::mount::mount(
+        Some(fs_type.as_str()),
+        mountpoint_c.as_c_str(),
+        Some(fs_type.as_str()),
+        flags,
+        Some("dir=/keys"),
+    )
+    .unwrap_or_else(|e| panic!("mount failed: {e}"));
+
+    let KEY_FILE: &str = "/mnt/key.txt";
```

Comment:
Double check this path matches the path in https://github.com/confidential-containers/guest-components/pull/385

Reply:
To boot/app enclave to run this need to match so if the e2e flow is working that means that the path is corresponding